Platform/STM/F4/CAN/bxcan_stmf4.cpp

```diff
@@ -39,6 +39,11 @@ void BxCanStmF4::ConfigureFilter(uint32_t filder_id_high, uint32_t filter_id_low
 }
 
 void BxCanStmF4::Start() {
+	tx_message_header_.IDE = CAN_ID_STD; // setting ID to Standard ID
+	tx_message_header_.StdId = 0x417; // Identifier (D = 4/ A = 1/ Q = (1)7)
```

Comment:
I do not find this comment too helpful

Reply:
It's pretty arcane reading it back, I was trying to make a meaningful identifier, suggestions?